src/main/java/pl/winiarczyk/CinemaEmployees/Manager.java

```diff
@@ -0,0 +1,31 @@
+package pl.winiarczyk.CinemaEmployees;
+
+public class Manager extends BaseEmployee {
+    final int salary = 5000;
+    int bonus = 500;
```

Comment:
to powinno być private + nie widzę w jaki sposób można zmienić ten bonus.

Treść zadania:
 `● Dodaj możliwość zmiany wysokości bonusu dla managera`